core/src/apps/homescreen/homescreen.py

```diff
@@ -54,6 +67,9 @@ def do_render(self) -> None:
         # homescreen with shifted avatar and text on bottom
         # Differs for each model
 
+        if not utils.usb_data_connected():
```

Comment:
please render a header instead.
`ui.header_error("NO USB CONNECTION")` might fit?
this error is the highest priority so it should be the first check in the list

Reply:
Changed.